buddyboss/buddyboss-platform
# PROD-649 "is one of" search mode issue for gender field in profile search - Fixed

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #3267  .

### How to test the changes in this Pull Request:

1. enable Profile Search
2. create a Gender field in your Profile Fields
3. Create a Profile Search based on Gender and select "is one of" search mode
4. Try to search in directory and see it works

### Proof Screenshots or Video

https://share.vidyard.com/watch/FvJ33j4EewTtLnMHaP7eAJ?

### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
